### PR TITLE
fix lnd config not using correct bitcoin rpc port

### PR DESCRIPTION
**Pull request by pwltr**
Fixes lnd connection problems for citadel-dev (regtest)

### CODE REVIEW

**Review by AaronDewes — APPROVED** [review]

**Comment by pwltr** [comment]
@AaronDewes I don't have write access so I can't merge this. Just tested this with mainnet and works as expected.

**Comment by AaronDewes** [comment]
You should have write access now, I configured permissions wrongly before, sorry

**Pull request merged**
